templates/iam_policy/ingest_step_function_policy.json.tpl

```diff
@@ -27,9 +27,8 @@
         "arn:aws:dynamodb:eu-west-2:${account_id}:table/${ingest_lock_table_name}",
         "arn:aws:dynamodb:eu-west-2:${account_id}:table/${ingest_lock_table_name}/index/${ingest_lock_table_group_id_gsi_name}",
         "arn:aws:states:eu-west-2:${account_id}:stateMachine:${ingest_sfn_name}",
-        "arn:aws:states:eu-west-2:${account_id}:execution:${ingest_sfn_name}/StagingCacheS3ObjectKeys:*",
-        "arn:aws:states:eu-west-2:${account_id}:execution:${ingest_sfn_name}:*",
-        "arn:aws:sns:eu-west-2:${account_id}:${notifications_topic_name}",
+        "arn:aws:states:eu-west-2:${account_id}:execution:intg-dr2-ingest/MapOverEachFolderId:*",
```

Comment:
1. Should `intg-dr2-ingest` not be `${ingest_sfn_name}`?
2. Just out of curiosity, what is this resource? The Distributed Map?

Reply:
1. Yep, thanks. Copy and paste fail.
2. It is yeah, if there's a failure inside the distributed map, it lets us do a redrive from that failure.